web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -1,27 +1,53 @@
-import { Link, routes } from '@redwoodjs/router'
+import { useState, useEffect } from 'react'
+import { Link, navigate ,routes } from '@redwoodjs/router'
 import { useAuth } from '@redwoodjs/auth'
 import { Flash } from '@redwoodjs/web'
 import Tooltip from '@material-ui/core/Tooltip';
 import { useQuery } from '@redwoodjs/web'
+import Popover from '@material-ui/core/Popover'
 import {getActiveClasses} from 'get-active-classes'
 
 export const QUERY = gql`
   query FIND_USER_BY_ID($id: String!) {
     user: user(id: $id) {
       id
       image
+      userName
+      name
     }
   }
 `
-
 import avatar from 'src/assets/harold.jpg'
 import Svg from 'src/components/Svg'
 import ImageUploader from 'src/components/ImageUploader'
 import logo from 'src/layouts/MainLayout/Logo_2.jpg'
 
-const MainLayout = ({ children }) => {
+const MainLayout = ({ children}) => {
   const { logIn, logOut, isAuthenticated, currentUser } = useAuth()
   const {data, loading} = useQuery(QUERY, {variables: {id: currentUser?.sub}})
+  console.log(data?.user?.name);
```

Comment:
Lets try and remove console logs after we're done using them.

Reply:
okay